.eslintrc.json

```diff
@@ -19,7 +19,7 @@
       "error",
       "element"
     ],
-    "valid-jsdoc": "warn"
+    "valid-jsdoc": "error"
```

Comment:
I think this was added because it's an `error` by default. If this is removed the JSDoc issues will still error, so we can remove this line to make this file a bit smaller. (It's part of the `plugin:@wordpress/eslint-plugin/recommended` rules already.)

Reply:
Needed to keep this to configure [`requireReturn`](https://eslint.org/docs/rules/valid-jsdoc#requirereturn)